{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 23,
   "id": "66775b8b-35bd-418e-bc01-53ec06e57a39",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'data': [{'id': '10.5061/dryad.0rxwdbs3r', 'type': 'dois', 'attributes': {'doi': '10.5061/dryad.0rxwdbs3r', 'identifiers': [], 'creators': [{'name': 'Del Rio, Miren', 'nameType': 'Personal', 'givenName': 'Miren', 'familyName': 'Del Rio', 'affiliation': ['Instituto Nacional de Investigación y Tecnología Agraria y Alimentaria'], 'nameIdentifiers': [{'schemeUri': 'https://orcid.org', 'nameIdentifier': 'https://orcid.org/0000-0001-7496-3713', 'nameIdentifierScheme': 'ORCID'}]}, {'name': 'Ruiz‐Peinado, Ricardo', 'nameType': 'Personal', 'givenName': 'Ricardo', 'familyName': 'Ruiz‐Peinado', 'affiliation': ['Instituto Nacional de Investigación y Tecnología Agraria y Alimentaria'], 'nameIdentifiers': []}, {'name': 'Holm, Stig‐Olof', 'nameType': 'Personal', 'givenName': 'Stig‐Olof', 'familyName': 'Holm', 'affiliation': ['Umeå University'], 'nameIdentifiers': []}, {'name': 'Jansons, Aris', 'nameType': 'Personal', 'givenName': 'Aris', 'familyName': 'Jansons', 'affiliation': ['Latvian State Forest Research Institute \"Silava\"'], 'nameIdentifiers': []}, {'name': 'Nord‐Larsen, Thomas', 'nameType': 'Personal', 'givenName': 'Thomas', 'familyName': 'Nord‐Larsen', 'affiliation': ['University of Copenhagen'], 'nameIdentifiers': []}, {'name': 'Verheyen, Kris', 'nameType': 'Personal', 'givenName': 'Kris', 'familyName': 'Verheyen', 'affiliation': ['Ghent University'], 'nameIdentifiers': [{'schemeUri': 'https://orcid.org', 'nameIdentifier': 'https://orcid.org/0000-0002-2067-9108', 'nameIdentifierScheme': 'ORCID'}]}, {'name': 'Bravo‐Oviedo, Andrés', 'nameType': 'Personal', 'givenName': 'Andrés', 'familyName': 'Bravo‐Oviedo', 'affiliation': ['Spanish National Research Council'], 'nameIdentifiers': [{'schemeUri': 'https://orcid.org', 'nameIdentifier': 'https://orcid.org/0000-0001-7036-7041', 'nameIdentifierScheme': 'ORCID'}]}, {'name': 'Pretzsch, Hans', 'nameType': 'Personal', 'givenName': 'Hans', 'familyName': 'Pretzsch', 'affiliation': ['Technical University Munich'], 'nameIdentifiers': [{'schemeUri': 'https://orcid.org', 'nameIdentifier': 'https://orcid.org/0000-0002-4958-1868', 'nameIdentifierScheme': 'ORCID'}]}, {'name': 'Ruiz-Peinado, Ricardo', 'nameType': 'Personal', 'givenName': 'Ricardo', 'familyName': 'Ruiz-Peinado', 'affiliation': ['Instituto Nacional de Investigación y Tecnología Agraria y Alimentaria'], 'nameIdentifiers': []}, {'name': 'Jactel, Hervé', 'nameType': 'Personal', 'givenName': 'Hervé', 'familyName': 'Jactel', 'affiliation': ['University of Bordeaux'], 'nameIdentifiers': [{'schemeUri': 'https://orcid.org', 'nameIdentifier': 'https://orcid.org/0000-0002-8106-5310', 'nameIdentifierScheme': 'ORCID'}]}, {'name': 'Coll, Lluís', 'nameType': 'Personal', 'givenName': 'Lluís', 'familyName': 'Coll', 'affiliation': ['University of Lleida'], 'nameIdentifiers': []}, {'name': 'Löf, Magnus', 'nameType': 'Personal', 'givenName': 'Magnus', 'familyName': 'Löf', 'affiliation': ['Swedish University of Agricultural Sciences'], 'nameIdentifiers': []}, {'name': 'Aldea, Jorge', 'nameType': 'Personal', 'givenName': 'Jorge', 'familyName': 'Aldea', 'affiliation': ['Swedish University of Agricultural Sciences'], 'nameIdentifiers': [{'schemeUri': 'https://orcid.org', 'nameIdentifier': 'https://orcid.org/0000-0003-2568-5192', 'nameIdentifierScheme': 'ORCID'}]}, {'name': 'Ammer, Christian', 'nameType': 'Personal', 'givenName': 'Christian', 'familyName': 'Ammer', 'affiliation': ['University of Göttingen'], 'nameIdentifiers': []}, {'name': 'Avdagić, Admir', 'nameType': 'Personal', 'givenName': 'Admir', 'familyName': 'Avdagić', 'affiliation': ['University of Sarajevo'], 'nameIdentifiers': []}, {'name': 'Barbeito, Ignacio', 'nameType': 'Personal', 'givenName': 'Ignacio', 'familyName': 'Barbeito', 'affiliation': ['University of British Columbia'], 'nameIdentifiers': []}, {'name': 'Bielak, Kamil', 'nameType': 'Personal', 'givenName': 'Kamil', 'familyName': 'Bielak', 'affiliation': ['Warsaw University of Life Sciences'], 'nameIdentifiers': []}, {'name': 'Bravo, Felipe', 'nameType': 'Personal', 'givenName': 'Felipe', 'familyName': 'Bravo', 'affiliation': ['University of Valladolid'], 'nameIdentifiers': []}, {'name': 'Brazaitis, Gediminas', 'nameType': 'Personal', 'givenName': 'Gediminas', 'familyName': 'Brazaitis', 'affiliation': ['Vytautas Magnus University'], 'nameIdentifiers': []}, {'name': 'Cerný, Jakub', 'nameType': 'Personal', 'givenName': 'Jakub', 'familyName': 'Cerný', 'affiliation': ['Forestry and Game Management Research Institute'], 'nameIdentifiers': [{'schemeUri': 'https://orcid.org', 'nameIdentifier': 'https://orcid.org/0000-0002-9954-1506', 'nameIdentifierScheme': 'ORCID'}]}, {'name': 'Collet, Catherine', 'nameType': 'Personal', 'givenName': 'Catherine', 'familyName': 'Collet', 'affiliation': ['University of Lorraine'], 'nameIdentifiers': []}, {'name': 'Condés, Sonia', 'nameType': 'Personal', 'givenName': 'Sonia', 'familyName': 'Condés', 'affiliation': ['Technical University of Madrid'], 'nameIdentifiers': []}, {'name': 'Drössler, Lars', 'nameType': 'Personal', 'givenName': 'Lars', 'familyName': 'Drössler', 'affiliation': ['Ilia State University'], 'nameIdentifiers': []}, {'name': 'Fabrika, Marek', 'nameType': 'Personal', 'givenName': 'Marek', 'familyName': 'Fabrika', 'affiliation': ['Technical University of Zvolen'], 'nameIdentifiers': []}, {'name': 'Heym, Michael', 'nameType': 'Personal', 'givenName': 'Michael', 'familyName': 'Heym', 'affiliation': ['Technical University Munich'], 'nameIdentifiers': []}, {'name': 'Hylen, Gro', 'nameType': 'Personal', 'givenName': 'Gro', 'familyName': 'Hylen', 'affiliation': ['Norwegian Institute of Bioeconomy Research'], 'nameIdentifiers': []}, {'name': 'Janssons, Aris', 'nameType': 'Personal', 'givenName': 'Aris', 'familyName': 'Janssons', 'affiliation': ['Latvian State Forest Research Institute \"Silava\"'], 'nameIdentifiers': []}, {'name': 'Kurylyak, Viktor', 'nameType': 'Personal', 'givenName': 'Viktor', 'familyName': 'Kurylyak', 'affiliation': ['Ukrainian National Forestry University'], 'nameIdentifiers': []}, {'name': 'Lombardi, Fabio', 'nameType': 'Personal', 'givenName': 'Fabio', 'familyName': 'Lombardi', 'affiliation': ['University of Reggio Calabria'], 'nameIdentifiers': []}, {'name': 'Matović, Bratislav', 'nameType': 'Personal', 'givenName': 'Bratislav', 'familyName': 'Matović', 'affiliation': ['University of East Sarajevo'], 'nameIdentifiers': []}, {'name': 'Metslaid, Marek', 'nameType': 'Personal', 'givenName': 'Marek', 'familyName': 'Metslaid', 'affiliation': ['Estonian University of Life Sciences'], 'nameIdentifiers': []}, {'name': 'Motta, Renzo', 'nameType': 'Personal', 'givenName': 'Renzo', 'familyName': 'Motta', 'affiliation': ['University of Turin'], 'nameIdentifiers': []}, {'name': 'Nord-Larsen, Thomas', 'nameType': 'Personal', 'givenName': 'Thomas', 'familyName': 'Nord-Larsen', 'affiliation': ['University of Copenhagen'], 'nameIdentifiers': []}, {'name': 'Nothdurft, Arne', 'nameType': 'Personal', 'givenName': 'Arne', 'familyName': 'Nothdurft', 'affiliation': ['University of Natural Resources and Life Sciences'], 'nameIdentifiers': []}, {'name': 'Den Ouden, Jan', 'nameType': 'Personal', 'givenName': 'Jan', 'familyName': 'Den Ouden', 'affiliation': ['Wageningen University & Research'], 'nameIdentifiers': []}, {'name': 'Pach, Maciej', 'nameType': 'Personal', 'givenName': 'Maciej', 'familyName': 'Pach', 'affiliation': ['University of Agriculture in Krakow'], 'nameIdentifiers': []}, {'name': 'Pardos, Marta', 'nameType': 'Personal', 'givenName': 'Marta', 'familyName': 'Pardos', 'affiliation': ['Instituto Nacional de Investigación y Tecnología Agraria y Alimentaria'], 'nameIdentifiers': []}, {'name': 'Poeydebat, Charlotte', 'nameType': 'Personal', 'givenName': 'Charlotte', 'familyName': 'Poeydebat', 'affiliation': ['University of Bordeaux'], 'nameIdentifiers': []}, {'name': 'Ponette, Quentin', 'nameType': 'Personal', 'givenName': 'Quentin', 'familyName': 'Ponette', 'affiliation': ['Université Catholique de Louvain'], 'nameIdentifiers': []}, {'name': 'Pérot, Tomas', 'nameType': 'Personal', 'givenName': 'Tomas', 'familyName': 'Pérot', 'affiliation': ['Institut National de la Recherche Agronomique'], 'nameIdentifiers': [{'schemeUri': 'https://orcid.org', 'nameIdentifier': 'https://orcid.org/0000-0002-0806-6819', 'nameIdentifierScheme': 'ORCID'}]}, {'name': 'Reventlow, Ditlev Otto Juel', 'nameType': 'Personal', 'givenName': 'Ditlev Otto Juel', 'familyName': 'Reventlow', 'affiliation': ['University of Copenhagen'], 'nameIdentifiers': []}, {'name': 'Sitko, Roman', 'nameType': 'Personal', 'givenName': 'Roman', 'familyName': 'Sitko', 'affiliation': ['Technical University of Zvolen'], 'nameIdentifiers': []}, {'name': 'Sramek, Vit', 'nameType': 'Personal', 'givenName': 'Vit', 'familyName': 'Sramek', 'affiliation': ['Forestry and Game Management Research Institute'], 'nameIdentifiers': []}, {'name': 'Steckel, Mathias', 'nameType': 'Personal', 'givenName': 'Mathias', 'familyName': 'Steckel', 'affiliation': ['Forst Baden-Württemberg (AöR)'], 'nameIdentifiers': []}, {'name': 'Svoboda, Miroslav', 'nameType': 'Personal', 'givenName': 'Miroslav', 'familyName': 'Svoboda', 'affiliation': ['Czech University of Life Sciences Prague'], 'nameIdentifiers': []}, {'name': 'Vospernik, Sonja', 'nameType': 'Personal', 'givenName': 'Sonja', 'familyName': 'Vospernik', 'affiliation': ['University of Natural Resources and Life Sciences'], 'nameIdentifiers': []}, {'name': 'Wolff, Barbara', 'nameType': 'Personal', 'givenName': 'Barbara', 'familyName': 'Wolff', 'affiliation': ['Hochschule für nachhaltige Entwicklung Eberswalde (HNEE), FG Waldinventur und Planung'], 'nameIdentifiers': []}, {'name': 'Zlatanov, Tzvetan', 'nameType': 'Personal', 'givenName': 'Tzvetan', 'familyName': 'Zlatanov', 'affiliation': ['Bulgarian Academy of Sciences'], 'nameIdentifiers': []}, {'name': 'Bravo-Oviedo, Andrés', 'nameType': 'Personal', 'givenName': 'Andrés', 'familyName': 'Bravo-Oviedo', 'affiliation': ['Spanish National Research Council'], 'nameIdentifiers': []}], 'titles': [{'title': 'Emerging stability of forest productivity by mixing two species buffers temperature destabilizing effect'}], 'publisher': 'Dryad', 'container': {}, 'publicationYear': 2022, 'subjects': [{'subject': 'Pinus sylvestris'}, {'subject': 'Fagus sylvatica'}, {'subject': 'Quercus petraea'}, {'subject': 'Picea abies'}, {'subject': 'Basal area growth'}, {'subject': 'Temporal stability'}, {'subject': 'overyielding'}, {'subject': 'climate effect'}, {'subject': 'species asynchrony'}, {'subject': 'additive effect'}, {'subject': 'forest ecosystems productivity'}, {'subject': 'mixed forests'}, {'subject': 'FOS: Agriculture, forestry, and fisheries'}], 'contributors': [], 'dates': [{'date': '2022', 'dateType': 'Issued'}], 'language': 'en', 'types': {'ris': 'DATA', 'bibtex': 'misc', 'citeproc': 'dataset', 'schemaOrg': 'Dataset', 'resourceType': 'dataset', 'resourceTypeGeneral': 'Dataset'}, 'relatedIdentifiers': [{'relationType': 'IsCitedBy', 'relatedIdentifier': '10.1111/1365-2664.14267', 'relatedIdentifierType': 'DOI'}], 'sizes': ['330072 bytes'], 'formats': [], 'version': '3', 'rightsList': [{'rights': 'Creative Commons Zero v1.0 Universal', 'rightsUri': 'https://creativecommons.org/publicdomain/zero/1.0/legalcode', 'schemeUri': 'https://spdx.org/licenses/', 'rightsIdentifier': 'cc0-1.0', 'rightsIdentifierScheme': 'SPDX'}], 'descriptions': [{'description': 'The increasing disturbances in monocultures around the world are testimony to their instability under global change. Many studies have claimed that temporal stability of productivity increase with species richness, although the ecological fundaments have mainly been investigated through diversity experiments. To adequately manage forest ecosystems, it is necessary to have a comprehensive understanding of the effect of mixing species on the temporal stability of productivity and the way in which this it is influenced by climate conditions across large geographical areas. Here, we used a unique dataset of 261 stands combining pure and two-species mixtures of four relevant tree species over a wide range of climate conditions in Europe to examine the effect of species mixing on the level and temporal stability of productivity. Structural equation modelling was employed to further explore the direct and indirect influence of climate, overyielding, species asynchrony and additive effect (i.e. temporal stability expected from the species growth in monospecific stands) on temporal stability in mixed forests. We showed that by adding only one tree species to monocultures, the level (overyielding: +6%) and stability (temporal stability: +12%) of stand growth increased significantly. We identified the key effect of temperature on destabilizing stand growth, which may be mitigated by mixing species. We further confirmed asynchrony as the main driver of temporal stability in mixed stands, through both the additive effect and species interactions, which modify between-species asynchrony in mixtures in comparison to monocultures. Synthesis and applications. This study highlights the emergent properties associated with mixing two-species, which result in resource efficient and temporally stable production systems. We reveal the negative impact of mean temperature on temporal stability of forest productivity and how the stabilizing effect of mixing two species can counterbalance this impact. The overyielding and temporal stability of growth addressed in this paper are essential for ecosystem services closely linked with the level and rhythm of forest growth. Our results underline that mixing two species can be a realistic and effective nature-based climate solution, which could contribute towards meeting EU climate target policies.', 'descriptionType': 'Abstract'}, {'description': 'The research unit is the forest stand. We used data from a total of 261 forest stands belonging to three triplet-transects across Europe. Each triplet consists of a plot established in a two species mixed stands, and two plots on the respective monospecific stands; the three stands are located close to each other under similar environmental conditions. The species composition of the mixtures changes in the three triplet-transects. The first transect covers monospecific and mixed stands of Fagus sylvatica and Pinus sylvestris (32 sites, 96 stands), the second of Quercus petraea and Pinus sylvestris (35 sites, 105 stands), and the third of Picea abies and Pinus sylvestris (20 sites, 60 stands). Plot sizes varies from 0-02 to 0.15 ha depending on stand density a local site characteristics. In each plot the diameter of all trees was measured, and two increment cores per tree were taken at a 1.3 m stem height in a sample of approximately 20 trees per species and plot. Annual ring widths were measured and cross-dated using standardized dendrochronological techniques. The studied period was 2000-2013 for the beech-pine transect and 2004-2017 for the oak-pine and spruce-pine transects (except in five triplets where the period was 2000-2013), the last year corresponding to triplet establishment. Using data from cored trees, tree diameter increment-diameter models were fitted by year, species and plot to estimate diameter increments of noncored trees for the studied period. Dead trees during the last 14 years were estimated using stumps, standing and lying dead trees, and their decomposition status. Based on measured tree diameters and annual diameter increments we estimated species and stand annual basal area (BA) and basal area growth (BAI), which conforms the dataset. Annual climate data were obtained from meteorological weather stations located in the proximity of each triplet (50 triplets). When local station data were not available, national digital climatic atlas data (24 triplets) or more general gridded data (13 triplets) were used (mostly CRU gridded database). For each triplet mean and standard deviation of annual precipitation (P) and mean annual temperature (T) for the studied period were calculated.', 'descriptionType': 'Methods'}, {'description': 'The are presented in an excel-table which can be open freely in different ways (open office, google docs, etc.)', 'descriptionType': 'Other'}], 'geoLocations': [], 'fundingReferences': [{'funderName': 'Ministerio de Ciencia, Innovación y Universidades', 'funderIdentifier': 'https://doi.org/10.13039/100014440', 'funderIdentifierType': 'Crossref Funder ID'}], 'url': 'http://datadryad.org/stash/dataset/doi:10.5061/dryad.0rxwdbs3r', 'contentUrl': None, 'metadataVersion': 0, 'schemaVersion': 'http://datacite.org/schema/kernel-4', 'source': 'mds', 'isActive': True, 'state': 'findable', 'reason': None, 'viewCount': 0, 'downloadCount': 0, 'referenceCount': 0, 'citationCount': 1, 'partCount': 0, 'partOfCount': 0, 'versionCount': 0, 'versionOfCount': 0, 'created': '2022-08-08T16:46:24Z', 'registered': '2022-08-08T16:46:25Z', 'published': None, 'updated': '2022-09-01T08:56:02Z'}, 'relationships': {'client': {'data': {'id': 'dryad.dryad', 'type': 'clients'}}}}, {'id': '10.25338/b8d61n', 'type': 'dois', 'attributes': {'doi': '10.25338/b8d61n', 'identifiers': [], 'creators': [{'name': 'Vasey, Dylan', 'nameType': 'Personal', 'givenName': 'Dylan', 'familyName': 'Vasey', 'affiliation': ['University of California, Davis'], 'nameIdentifiers': [{'schemeUri': 'https://orcid.org', 'nameIdentifier': 'https://orcid.org/0000-0002-2182-4733', 'nameIdentifierScheme': 'ORCID'}]}, {'name': 'Cowgill, Eric', 'nameType': 'Personal', 'givenName': 'Eric', 'familyName': 'Cowgill', 'affiliation': ['University of California, Davis'], 'nameIdentifiers': []}, {'name': 'Roeske, Sarah', 'nameType': 'Personal', 'givenName': 'Sarah', 'familyName': 'Roeske', 'affiliation': ['University of California, Davis'], 'nameIdentifiers': []}, {'name': 'Niemi, Nathan', 'nameType': 'Personal', 'givenName': 'Nathan', 'familyName': 'Niemi', 'affiliation': ['University of Michigan'], 'nameIdentifiers': []}, {'name': 'Godoladze, Tea', 'nameType': 'Personal', 'givenName': 'Tea', 'familyName': 'Godoladze', 'affiliation': ['Ilia State University'], 'nameIdentifiers': []}, {'name': 'Skhirtladze, Irakli', 'nameType': 'Personal', 'givenName': 'Irakli', 'familyName': 'Skhirtladze', 'affiliation': ['Ilia State University'], 'nameIdentifiers': []}, {'name': 'Gogoladze, Salome', 'nameType': 'Personal', 'givenName': 'Salome', 'familyName': 'Gogoladze', 'affiliation': ['Ilia State University'], 'nameIdentifiers': []}], 'titles': [{'title': 'Evolution of the Greater Caucasus basement and formation of the Main Caucasus Thrust, Georgia'}], 'publisher': 'Dryad', 'container': {}, 'publicationYear': 2020, 'subjects': [{'subject': 'Main Caucasus Thrust'}], 'contributors': [], 'dates': [{'date': '2020', 'dateType': 'Issued'}], 'language': 'en', 'types': {'ris': 'DATA', 'bibtex': 'misc', 'citeproc': 'dataset', 'schemaOrg': 'Dataset', 'resourceType': 'dataset', 'resourceTypeGeneral': 'Dataset'}, 'relatedIdentifiers': [], 'sizes': ['7147534 bytes'], 'formats': [], 'version': '4', 'rightsList': [{'rights': 'Creative Commons Zero v1.0 Universal', 'rightsUri': 'https://creativecommons.org/publicdomain/zero/1.0/legalcode', 'schemeUri': 'https://spdx.org/licenses/', 'rightsIdentifier': 'cc0-1.0', 'rightsIdentifierScheme': 'SPDX'}], 'descriptions': [{'description': 'Along the northern margin of the Arabia-Eurasia collision zone in the western Greater Caucasus, the Main Caucasus Thrust (MCT) juxtaposes Paleozoic crystalline basement to the north against Mesozoic metasedimentary and volcaniclastic rocks to the south. The MCT is commonly assumed to be the trace of an active plate-boundary scale structure that accommodates Arabia-Eurasia convergence, but field data supporting this interpretation are equivocal. Here we investigate the deformation history of the rocks juxtaposed across the MCT in Georgia using field observations, microstructural analysis, U-Pb and 40Ar/39Ar geochronology, and 40Ar/39Ar and (U-Th)/He thermochronology. Zircon U-Pb analyses show that Greater Caucasus crystalline rocks formed in the early Paleozoic on the margin of Gondwana. Low pressure/temperature amphibolite-facies metamorphism of these metasedimentary rocks and associated plutonism likely took place during Carboniferous accretion onto the Laurussian margin, as indicated by igneous and metamorphic zircon U-Pb ages of ~330-310 Ma. 40Ar/39Ar ages of ~190-135 Ma from muscovite in a greenschist facies shear zone indicate that the MCT likely developed during Mesozoic inversion and/or rifting of the Caucasus Basin. A Mesozoic 40Ar/39Ar biotite age with release spectra indicating partial resetting and Cenozoic (&lt;40 Ma) apatite and zircon (U-Th)/He ages imply at least ~5-8 km of Greater Caucasus basement exhumation since ~10 Ma in response to Arabia-Eurasia collision. Cenozoic reactivation of the MCT may have accommodated a fraction of this exhumation. However, Cenozoic zircon (U-Th)/He ages in both the hanging wall and footwall of the MCT require partitioning a substantial component of this deformation onto structures to the south.', 'descriptionType': 'Abstract'}], 'geoLocations': [], 'fundingReferences': [{'funderName': 'National Science Foundation', 'awardNumber': '0810285', 'funderIdentifier': 'https://doi.org/10.13039/100000001', 'funderIdentifierType': 'Crossref Funder ID'}, {'funderName': 'National Science Foundation', 'awardNumber': '1524631', 'funderIdentifier': 'https://doi.org/10.13039/100000001', 'funderIdentifierType': 'Crossref Funder ID'}, {'funderName': 'National Science Foundation', 'awardNumber': '0810067', 'funderIdentifier': 'https://doi.org/10.13039/100000001', 'funderIdentifierType': 'Crossref Funder ID'}, {'funderName': 'National Science Foundation', 'awardNumber': '1524304', 'funderIdentifier': 'https://doi.org/10.13039/100000001', 'funderIdentifierType': 'Crossref Funder ID'}, {'funderName': 'Geological Society of America', 'funderIdentifier': 'https://doi.org/10.13039/100005720', 'funderIdentifierType': 'Crossref Funder ID'}, {'funderName': 'Sigma Xi', 'funderIdentifier': 'https://doi.org/10.13039/100011084', 'funderIdentifierType': 'Crossref Funder ID'}, {'funderName': 'University of California, Davis', 'funderIdentifier': 'https://doi.org/10.13039/100007707', 'funderIdentifierType': 'Crossref Funder ID'}], 'url': 'http://datadryad.org/stash/dataset/doi:10.25338/B8D61N', 'contentUrl': None, 'metadataVersion': 3, 'schemaVersion': 'http://datacite.org/schema/kernel-4', 'source': 'mds', 'isActive': True, 'state': 'findable', 'reason': None, 'viewCount': 35, 'downloadCount': 7, 'referenceCount': 0, 'citationCount': 0, 'partCount': 0, 'partOfCount': 0, 'versionCount': 0, 'versionOfCount': 0, 'created': '2020-03-05T15:54:53Z', 'registered': '2020-03-05T15:54:54Z', 'published': None, 'updated': '2022-06-03T21:57:18Z'}, 'relationships': {'client': {'data': {'id': 'cdl.ucd', 'type': 'clients'}}}}, {'id': '10.5061/dryad.6d57c', 'type': 'dois', 'attributes': {'doi': '10.5061/dryad.6d57c', 'identifiers': [], 'creators': [{'name': 'Jactel, Herve', 'nameType': 'Personal', 'givenName': 'Herve', 'familyName': 'Jactel', 'affiliation': ['University of Bordeaux'], 'nameIdentifiers': []}, {'name': 'Gritti, Emmanuel S.', 'nameType': 'Personal', 'givenName': 'Emmanuel S.', 'familyName': 'Gritti', 'affiliation': ['Montpellier SupAgro'], 'nameIdentifiers': []}, {'name': 'Drössler, Lars', 'nameType': 'Personal', 'givenName': 'Lars', 'familyName': 'Drössler', 'affiliation': ['Ilia State University'], 'nameIdentifiers': []}, {'name': 'Forrester, David I.', 'nameType': 'Personal', 'givenName': 'David I.', 'familyName': 'Forrester', 'affiliation': ['Swiss Federal Institute for Forest, Snow and Landscape Research'], 'nameIdentifiers': []}, {'name': 'Mason, William L.', 'nameType': 'Personal', 'givenName': 'William L.', 'familyName': 'Mason', 'affiliation': ['Forest Research'], 'nameIdentifiers': []}, {'name': 'Morin, Xavier', 'nameType': 'Personal', 'givenName': 'Xavier', 'familyName': 'Morin', 'affiliation': ['French National Centre for Scientific Research'], 'nameIdentifiers': []}, {'name': 'Pretzsch, Hans', 'nameType': 'Personal', 'givenName': 'Hans', 'familyName': 'Pretzsch', 'affiliation': ['Technical University Munich'], 'nameIdentifiers': []}, {'name': 'Castagneyrol, Bastien', 'nameType': 'Personal', 'givenName': 'Bastien', 'familyName': 'Castagneyrol', 'affiliation': ['University of Bordeaux'], 'nameIdentifiers': []}], 'titles': [{'title': 'Data from: Positive biodiversity-productivity relationships in forests: climate matters'}], 'publisher': 'Dryad', 'container': {}, 'publicationYear': 2018, 'subjects': [{'subject': 'overyielding'}, {'subject': 'nowadays'}], 'contributors': [], 'dates': [{'date': '2020-07-05T00:00:00Z', 'dateType': 'Available'}, {'date': '2018', 'dateType': 'Issued'}], 'language': 'en', 'types': {'ris': 'DATA', 'bibtex': 'misc', 'citeproc': 'dataset', 'schemaOrg': 'Dataset', 'resourceType': 'dataset', 'resourceTypeGeneral': 'Dataset'}, 'relatedIdentifiers': [{'relationType': 'IsCitedBy', 'relatedIdentifier': '10.1098/rsbl.2017.0747', 'relatedIdentifierType': 'DOI'}], 'sizes': ['10838 bytes'], 'formats': [], 'version': '1', 'rightsList': [{'rights': 'Creative Commons Zero v1.0 Universal', 'rightsUri': 'https://creativecommons.org/publicdomain/zero/1.0/legalcode', 'schemeUri': 'https://spdx.org/licenses/', 'rightsIdentifier': 'cc0-1.0', 'rightsIdentifierScheme': 'SPDX'}], 'descriptions': [{'description': 'While it is widely acknowledged that forest biodiversity contributes to climate change mitigation through improved carbon sequestration, conversely how climate affects tree species diversity - forest productivity relationships is still poorly understood. We combined the results of long-term experiments where forest mixtures and corresponding monocultures were compared on the same site to estimate the yield of mixed-species stands at a global scale, and its response to climatic factors. We found positive mixture effects on productivity using a meta-analysis of 126 cases studies established at 60 sites spread across five continents. Overall, the productivity of mixed-species forests was 15% greater than the average of their component monocultures, and not statistically lower than productivity of the best component monoculture. Productivity gains in mixed-species stands were not affected by tree age or stand species composition but significantly increased with local precipitation. The results should guide better use of tree species combinations in managed forests and suggest that increased drought severity under climate change might reduce the atmospheric carbon sequestration capacity of natural forests.', 'descriptionType': 'Abstract'}, {'description': 'Jactel-Forest-Diversity-Productivity-Climate-meta-analysis-dataSummary of data included in the meta-analyses', 'descriptionType': 'Other'}], 'geoLocations': [{'geoLocationPlace': 'world wide'}], 'fundingReferences': [{'funderName': 'National Science Foundation', 'awardNumber': 'European Commission GA no.226299', 'funderIdentifier': 'https://doi.org/10.13039/100000001', 'funderIdentifierType': 'Crossref Funder ID'}], 'url': 'http://datadryad.org/stash/dataset/doi:10.5061/dryad.6d57c', 'contentUrl': None, 'metadataVersion': 21, 'schemaVersion': 'http://datacite.org/schema/kernel-4', 'source': 'mds', 'isActive': True, 'state': 'findable', 'reason': None, 'viewCount': 174, 'downloadCount': 69, 'referenceCount': 1, 'citationCount': 1, 'partCount': 1, 'partOfCount': 0, 'versionCount': 0, 'versionOfCount': 0, 'created': '2018-03-20T17:41:24Z', 'registered': '2018-03-20T17:41:25Z', 'published': None, 'updated': '2022-06-01T13:35:02Z'}, 'relationships': {'client': {'data': {'id': 'dryad.dryad', 'type': 'clients'}}}}, {'id': '10.5061/dryad.d08fb', 'type': 'dois', 'attributes': {'doi': '10.5061/dryad.d08fb', 'identifiers': [], 'creators': [{'name': 'Hierro, José L.', 'nameType': 'Personal', 'givenName': 'José L.', 'familyName': 'Hierro', 'affiliation': ['National University of La Pampa'], 'nameIdentifiers': []}, {'name': 'Khetsuriani, Liana', 'nameType': 'Personal', 'givenName': 'Liana', 'familyName': 'Khetsuriani', 'affiliation': ['Ilia State University'], 'nameIdentifiers': []}, {'name': 'Andonian, Krikor', 'nameType': 'Personal', 'givenName': 'Krikor', 'familyName': 'Andonian', 'affiliation': ['De Anza College'], 'nameIdentifiers': []}, {'name': 'Eren, Özkan', 'nameType': 'Personal', 'givenName': 'Özkan', 'familyName': 'Eren', 'affiliation': ['Adnan Menderes University'], 'nameIdentifiers': []}, {'name': 'Villarreal, Diego', 'nameType': 'Personal', 'givenName': 'Diego', 'familyName': 'Villarreal', 'affiliation': ['National University of La Pampa'], 'nameIdentifiers': []}, {'name': 'Janoian, Grigor', 'nameType': 'Personal', 'givenName': 'Grigor', 'familyName': 'Janoian', 'affiliation': ['American University of Armenia'], 'nameIdentifiers': []}, {'name': 'Reinhart, Kurt O.', 'nameType': 'Personal', 'givenName': 'Kurt O.', 'familyName': 'Reinhart', 'affiliation': ['University of Montana'], 'nameIdentifiers': []}, {'name': 'Callaway, Ragan M.', 'nameType': 'Personal', 'givenName': 'Ragan M.', 'familyName': 'Callaway', 'affiliation': ['University of Montana'], 'nameIdentifiers': []}], 'titles': [{'title': 'Data from: The importance of factors controlling species abundance and distribution varies in native and non-native species.'}], 'publisher': 'Dryad', 'container': {}, 'publicationYear': 2016, 'subjects': [{'subject': 'community resilience'}, {'subject': 'Biogeographical field experiments'}, {'subject': 'Centaurea solstitialis'}], 'contributors': [], 'dates': [{'date': '2020-06-27T00:00:00Z', 'dateType': 'Available'}, {'date': '2016', 'dateType': 'Issued'}], 'language': 'en', 'types': {'ris': 'DATA', 'bibtex': 'misc', 'citeproc': 'dataset', 'schemaOrg': 'Dataset', 'resourceType': 'dataset', 'resourceTypeGeneral': 'Dataset'}, 'relatedIdentifiers': [{'relationType': 'IsCitedBy', 'relatedIdentifier': '10.1111/ecog.02633', 'relatedIdentifierType': 'DOI'}], 'sizes': ['244835 bytes'], 'formats': [], 'version': '1', 'rightsList': [{'rights': 'Creative Commons Zero v1.0 Universal', 'rightsUri': 'https://creativecommons.org/publicdomain/zero/1.0/legalcode', 'schemeUri': 'https://spdx.org/licenses/', 'rightsIdentifier': 'cc0-1.0', 'rightsIdentifierScheme': 'SPDX'}], 'descriptions': [{'description': 'How variation in factors controlling species abundance and distribution between native and non-native ranges compares to that within ranges remains poorly understood. We used a globally distributed ruderal, Centaurea solstitialis (Centaurea), to explore the possibility that the importance of those factors exhibits great variation between and within ranges. To test our hypothesis, we established seed addition experiments with soil disturbance (turnover and control) and biocide (fungicides, insecticide, and control) treatments in two regions within native (the Caucasus and south-western Turkey) and non-native (the western United States – US – and central Argentina) distributions. Also, we estimated the rate of vegetation recovery after disturbance (resilience) and related it to Centaurea density in experimental plots. Disturbance strongly increased Centaurea density in all regions. Density was similar between the native Caucasus and non-native Argentina and much greater in those regions than in the native Turkey and non-native US in biocide-free plots. Fungicides had positive effects on density in the US and negative ones in the Caucasus and Argentina, resulting in no differences between those three regions and greater density in the US than Turkey. Insecticide applications promoted Centaurea density in Turkey and Argentina, but inter-regional comparisons of density in treated plots were comparable to those in biocide-free plots. Overall, plants were smaller and less fecund in Turkey than the other regions, except the US. The greatest fungal attack was documented in Turkey, and herbivory was stronger there and in Argentina than in the Caucasus and US. The resilience of the local community explained a large proportion of variation in Centaurea density. These results support our hypothesis, and reveal that the speed at which competition is re-gained after disturbance may influence global variation in Centaurea abundance. Because many ruderals exhibit native and non-native distributions, our results are likely to be generalized to other systems.', 'descriptionType': 'Abstract'}, {'description': 'Data from Hierro et al. 2016 EcographyThis is an Excel file, where data from our study is organized in different sheets.', 'descriptionType': 'Other'}], 'geoLocations': [{'geoLocationPlace': 'Southwestern Turkey'}, {'geoLocationPlace': 'Central Argentina'}, {'geoLocationPlace': 'Western US'}, {'geoLocationPlace': 'Caucasus'}], 'fundingReferences': [], 'url': 'http://datadryad.org/stash/dataset/doi:10.5061/dryad.d08fb', 'contentUrl': None, 'metadataVersion': 25, 'schemaVersion': 'http://datacite.org/schema/kernel-4', 'source': 'mds', 'isActive': True, 'state': 'findable', 'reason': None, 'viewCount': 71, 'downloadCount': 14, 'referenceCount': 1, 'citationCount': 1, 'partCount': 1, 'partOfCount': 0, 'versionCount': 0, 'versionOfCount': 0, 'created': '2016-07-28T05:39:42Z', 'registered': '2016-07-28T05:39:43Z', 'published': None, 'updated': '2022-06-01T01:48:26Z'}, 'relationships': {'client': {'data': {'id': 'dryad.dryad', 'type': 'clients'}}}}, {'id': '10.5061/dryad.f0k4j', 'type': 'dois', 'attributes': {'doi': '10.5061/dryad.f0k4j', 'identifiers': [], 'creators': [{'name': 'Urushadze, Lela', 'nameType': 'Personal', 'givenName': 'Lela', 'familyName': 'Urushadze', 'affiliation': ['National Center for Disease Control and Public Health', 'Ilia State University'], 'nameIdentifiers': []}, {'name': 'Bai, Ying', 'nameType': 'Personal', 'givenName': 'Ying', 'familyName': 'Bai', 'affiliation': ['Centers for Disease Control and Prevention'], 'nameIdentifiers': []}, {'name': 'Osikowicz, Lynn', 'nameType': 'Personal', 'givenName': 'Lynn', 'familyName': 'Osikowicz', 'affiliation': ['Centers for Disease Control and Prevention'], 'nameIdentifiers': []}, {'name': 'McKee, Clifton', 'nameType': 'Personal', 'givenName': 'Clifton', 'familyName': 'McKee', 'affiliation': ['Colorado State University', 'Centers for Disease Control and Prevention'], 'nameIdentifiers': []}, {'name': 'Sidamonidze, Ketevan', 'nameType': 'Personal', 'givenName': 'Ketevan', 'familyName': 'Sidamonidze', 'affiliation': ['National Center for Disease Control and Public Health'], 'nameIdentifiers': []}, {'name': 'Putkaradze, Davit', 'nameType': 'Personal', 'givenName': 'Davit', 'familyName': 'Putkaradze', 'affiliation': ['National Center for Disease Control and Public Health'], 'nameIdentifiers': []}, {'name': 'Imnadze, Paata', 'nameType': 'Personal', 'givenName': 'Paata', 'familyName': 'Imnadze', 'affiliation': ['National Center for Disease Control and Public Health'], 'nameIdentifiers': []}, {'name': 'Kandaurov, Andrei', 'nameType': 'Personal', 'givenName': 'Andrei', 'familyName': 'Kandaurov', 'affiliation': ['Ilia State University'], 'nameIdentifiers': []}, {'name': 'Kuzmin, Ivan', 'nameType': 'Personal', 'givenName': 'Ivan', 'familyName': 'Kuzmin', 'affiliation': [], 'nameIdentifiers': []}, {'name': 'Kosoy, Michael', 'nameType': 'Personal', 'givenName': 'Michael', 'familyName': 'Kosoy', 'affiliation': ['Centers for Disease Control and Prevention'], 'nameIdentifiers': []}], 'titles': [{'title': 'Data from: Prevalence, diversity, and host associations of Bartonella strains in bats from Georgia (Caucasus)'}], 'publisher': 'Dryad', 'container': {}, 'publicationYear': 2018, 'subjects': [{'subject': 'phylogenetic analysis'}, {'subject': 'Bartonella'}, {'subject': 'Dogs'}, {'subject': 'sequence alignment'}, {'subject': 'Genetic loci'}, {'subject': 'Sequence analysis'}, {'subject': 'host-pathogen interactions'}], 'contributors': [], 'dates': [{'date': '2018-03-20T00:00:00Z', 'dateType': 'Available'}, {'date': '2018', 'dateType': 'Issued'}], 'language': 'en', 'types': {'ris': 'DATA', 'bibtex': 'misc', 'citeproc': 'dataset', 'schemaOrg': 'Dataset', 'resourceType': 'dataset', 'resourceTypeGeneral': 'Dataset'}, 'relatedIdentifiers': [{'relationType': 'IsCitedBy', 'relatedIdentifier': '10.1371/journal.pntd.0005428', 'relatedIdentifierType': 'DOI'}], 'sizes': ['53298271 bytes'], 'formats': [], 'version': '1', 'rightsList': [{'rights': 'Creative Commons Zero v1.0 Universal', 'rightsUri': 'https://creativecommons.org/publicdomain/zero/1.0/legalcode', 'schemeUri': 'https://spdx.org/licenses/', 'rightsIdentifier': 'cc0-1.0', 'rightsIdentifierScheme': 'SPDX'}], 'descriptions': [{'description': 'Bartonella infections were investigated in seven species of bats from four regions of the Republic of Georgia. Of the 236 bats that were captured, 212 (90%) specimens were tested for Bartonella infection. Colonies identified as Bartonella were isolated from 105 (49.5%) of 212 bats Phylogenetic analysis based on sequence variation of the gltA gene differentiated 22 unique Bartonella genogroups. Genetic distances between these diverse genogroups were at the level of those observed between different Bartonella species described previously. Twenty-one reference strains from 19 representative genogroups were characterized using four additional genetic markers. Host specificity to bat genera or families was reported for several Bartonella genogroups. Some Bartonella genotypes found in bats clustered with those identified in dogs from Thailand and humans from Poland.', 'descriptionType': 'Abstract'}, {'description': 'Urushadze et al (2017) PLOS NTDData files used in Urushadze et al (2017) PLOS NTD \"Prevalence, Diversity, and Host Associations of Bartonella Strains in Bats from Georgia (Caucasus).\" Includes Excel tables containing data on collected bat specimens, bacterial isolates, sequences, and statistical analyses, Excel tables containing data on phylogenetic model selection, phylogenetic trees in Nexus format, parameter log files from Bayesian phylogenetic analyses, and supplementary documents (Appendix and supplementary text).', 'descriptionType': 'Other'}], 'geoLocations': [], 'fundingReferences': [], 'url': 'http://datadryad.org/stash/dataset/doi:10.5061/dryad.f0k4j', 'contentUrl': None, 'metadataVersion': 18, 'schemaVersion': 'http://datacite.org/schema/kernel-4', 'source': 'mds', 'isActive': True, 'state': 'findable', 'reason': None, 'viewCount': 135, 'downloadCount': 34, 'referenceCount': 1, 'citationCount': 2, 'partCount': 2, 'partOfCount': 0, 'versionCount': 0, 'versionOfCount': 0, 'created': '2017-03-20T15:27:14Z', 'registered': '2017-03-20T15:27:15Z', 'published': None, 'updated': '2022-05-31T21:32:09Z'}, 'relationships': {'client': {'data': {'id': 'dryad.dryad', 'type': 'clients'}}}}, {'id': '10.5061/dryad.cr1496b', 'type': 'dois', 'attributes': {'doi': '10.5061/dryad.cr1496b', 'identifiers': [], 'creators': [{'name': 'Barlow, Axel', 'nameType': 'Personal', 'givenName': 'Axel', 'familyName': 'Barlow', 'affiliation': ['University of Potsdam'], 'nameIdentifiers': []}, {'name': 'Cahill, James A.', 'nameType': 'Personal', 'givenName': 'James A.', 'familyName': 'Cahill', 'affiliation': ['University of California, Santa Cruz'], 'nameIdentifiers': []}, {'name': 'Hartmann, Stefanie', 'nameType': 'Personal', 'givenName': 'Stefanie', 'familyName': 'Hartmann', 'affiliation': ['University of Potsdam'], 'nameIdentifiers': []}, {'name': 'Theunert, Christoph', 'nameType': 'Personal', 'givenName': 'Christoph', 'familyName': 'Theunert', 'affiliation': ['University of California, Berkeley'], 'nameIdentifiers': []}, {'name': 'Xenikoudakis, Georgios', 'nameType': 'Personal', 'givenName': 'Georgios', 'familyName': 'Xenikoudakis', 'affiliation': ['University of Potsdam'], 'nameIdentifiers': []}, {'name': 'Fortes, Gloria G.', 'nameType': 'Personal', 'givenName': 'Gloria G.', 'familyName': 'Fortes', 'affiliation': ['University of Potsdam'], 'nameIdentifiers': []}, {'name': 'Paijmans, Johanna L. A.', 'nameType': 'Personal', 'givenName': 'Johanna L. A.', 'familyName': 'Paijmans', 'affiliation': ['University of Potsdam'], 'nameIdentifiers': []}, {'name': 'Rabeder, Gernot', 'nameType': 'Personal', 'givenName': 'Gernot', 'familyName': 'Rabeder', 'affiliation': ['University of Vienna'], 'nameIdentifiers': []}, {'name': 'Frischauf, Christine', 'nameType': 'Personal', 'givenName': 'Christine', 'familyName': 'Frischauf', 'affiliation': ['University of Vienna'], 'nameIdentifiers': []}, {'name': \"Grandal-D'Anglade, Aurora\", 'nameType': 'Personal', 'givenName': 'Aurora', 'familyName': \"Grandal-D'Anglade\", 'affiliation': ['University of A Coruña'], 'nameIdentifiers': []}, {'name': 'García-Vázquez, Ana', 'nameType': 'Personal', 'givenName': 'Ana', 'familyName': 'García-Vázquez', 'affiliation': ['University of A Coruña'], 'nameIdentifiers': []}, {'name': 'Murtskhvaladze, Marine', 'nameType': 'Personal', 'givenName': 'Marine', 'familyName': 'Murtskhvaladze', 'affiliation': ['Ilia State University'], 'nameIdentifiers': []}, {'name': 'Saarma, Urmas', 'nameType': 'Personal', 'givenName': 'Urmas', 'familyName': 'Saarma', 'affiliation': ['University of Tartu'], 'nameIdentifiers': []}, {'name': 'Anijalg, Peeter', 'nameType': 'Personal', 'givenName': 'Peeter', 'familyName': 'Anijalg', 'affiliation': ['University of Tartu'], 'nameIdentifiers': []}, {'name': 'Skrbinšek, Tomaž', 'nameType': 'Personal', 'givenName': 'Tomaž', 'familyName': 'Skrbinšek', 'affiliation': ['University of Ljubljana'], 'nameIdentifiers': []}, {'name': 'Bertorelle, Giorgio', 'nameType': 'Personal', 'givenName': 'Giorgio', 'familyName': 'Bertorelle', 'affiliation': ['University of Ferrara'], 'nameIdentifiers': []}, {'name': 'Gasparian, Boris', 'nameType': 'Personal', 'givenName': 'Boris', 'familyName': 'Gasparian', 'affiliation': ['National Academy of Sciences of Armenia'], 'nameIdentifiers': []}, {'name': 'Bar-Oz, Guy', 'nameType': 'Personal', 'givenName': 'Guy', 'familyName': 'Bar-Oz', 'affiliation': ['University of Haifa'], 'nameIdentifiers': []}, {'name': 'Pinhasi, Ron', 'nameType': 'Personal', 'givenName': 'Ron', 'familyName': 'Pinhasi', 'affiliation': ['University College Dublin'], 'nameIdentifiers': []}, {'name': 'Slatkin, Montgomery', 'nameType': 'Personal', 'givenName': 'Montgomery', 'familyName': 'Slatkin', 'affiliation': ['University of California, Berkeley'], 'nameIdentifiers': []}, {'name': 'Dalén, Love', 'nameType': 'Personal', 'givenName': 'Love', 'familyName': 'Dalén', 'affiliation': ['Swedish Museum of Natural History'], 'nameIdentifiers': []}, {'name': 'Shapiro, Beth', 'nameType': 'Personal', 'givenName': 'Beth', 'familyName': 'Shapiro', 'affiliation': ['University of California, Santa Cruz'], 'nameIdentifiers': []}, {'name': 'Hofreiter, Michael', 'nameType': 'Personal', 'givenName': 'Michael', 'familyName': 'Hofreiter', 'affiliation': ['University of Potsdam'], 'nameIdentifiers': []}], 'titles': [{'title': 'Data from: Partial genomic survival of cave bears in living brown bears'}], 'publisher': 'Dryad', 'container': {}, 'publicationYear': 2019, 'subjects': [{'subject': 'Ursus ingressus'}, {'subject': 'Late Pleistocene'}, {'subject': 'Ursus arctos'}, {'subject': 'Ursus kudarensis'}, {'subject': 'Ursus thibetanus'}, {'subject': 'haploidised fasta sequences'}, {'subject': 'Ursus spelaeus'}, {'subject': 'Holocene'}, {'subject': 'Tremarctos ornatus'}, {'subject': 'Ursus maritimus'}, {'subject': 'Ursus spelaeus eremus'}], 'contributors': [], 'dates': [{'date': '2019-07-12T00:00:00Z', 'dateType': 'Available'}, {'date': '2019', 'dateType': 'Issued'}], 'language': 'en', 'types': {'ris': 'DATA', 'bibtex': 'misc', 'citeproc': 'dataset', 'schemaOrg': 'Dataset', 'resourceType': 'dataset', 'resourceTypeGeneral': 'Dataset'}, 'relatedIdentifiers': [{'relationType': 'IsCitedBy', 'relatedIdentifier': '10.1038/s41559-018-0654-8', 'relatedIdentifierType': 'DOI'}], 'sizes': ['6331386741 bytes'], 'formats': [], 'version': '1', 'rightsList': [{'rights': 'Creative Commons Zero v1.0 Universal', 'rightsUri': 'https://creativecommons.org/publicdomain/zero/1.0/legalcode', 'schemeUri': 'https://spdx.org/licenses/', 'rightsIdentifier': 'cc0-1.0', 'rightsIdentifierScheme': 'SPDX'}], 'descriptions': [{'description': 'Although many large mammal species went extinct at the end of the Pleistocene epoch, their DNA may persist due to past episodes of interspecies admixture. However, direct empirical evidence of the persistence of ancient alleles remains scarce. Here, we present multifold coverage genomic data from four Late Pleistocene cave bears (Ursus spelaeus complex) and show that cave bears hybridized with brown bears (Ursus arctos) during the Pleistocene. We develop an approach to assess both the directionality and relative timing of gene flow. We find that segments of cave bear DNA still persist in the genomes of living brown bears, with cave bears contributing 0.9 to 2.4% of the genomes of all brown bears investigated. Our results show that even though extinction is typically considered as absolute, following admixture, fragments of the gene pool of extinct species can survive for tens of thousands of years in the genomes of extant recipient species.', 'descriptionType': 'Abstract'}, {'description': '191Y_arctos_Slovenia_haploidised_fastaHaploidised fasta sequence generated by mapping Illumina short reads to the reference genome assembly of the giant panda, then randomly selecting a single high quality nucleotide from the read stack for each position of the reference genome. See the original publication for full details. The raw sequencing data is also available from the European Nucleotide Archive. Note that this file will contain abundant errors in comparison to a consensus base call from high coverage data.191Y_rep1_all.fa.gz235_arctos_Russia_haploidised_fastaHaploidised fasta sequence generated by mapping Illumina short reads to the reference genome assembly of the giant panda, then randomly selecting a single high quality nucleotide from the read stack for each position of the reference genome. See the original publication for full details. The raw sequencing data is also available from the European Nucleotide Archive. Note that this file will contain abundant errors in comparison to a consensus base call from high coverage data.235_rep1_all.fa.gzAdm1_arctos_Admiralty_haploidised_fastaHaploidised fasta sequence generated by mapping Illumina short reads to the reference genome assembly of the giant panda, then randomly selecting a single high quality nucleotide from the read stack for each position of the reference genome. See the original publication for full details. The raw sequencing data is also available from the European Nucleotide Archive. Note that this file will contain abundant errors in comparison to a consensus base call from high coverage data.Adm1_rep1_all.fa.gzDen_arctos_Denali_haploidised_fastaHaploidised fasta sequence generated by mapping Illumina short reads to the reference genome assembly of the giant panda, then randomly selecting a single high quality nucleotide from the read stack for each position of the reference genome. See the original publication for full details. The raw sequencing data is also available from the European Nucleotide Archive. Note that this file will contain abundant errors in comparison to a consensus base call from high coverage data.Den_rep1_all.fa.gzE-VD-1838_spelaeus_Spain_haploidised_fastaHaploidised fasta sequence generated by mapping Illumina short reads to the reference genome assembly of the giant panda, then randomly selecting a single high quality nucleotide from the read stack for each position of the reference genome. See the original publication for full details. The raw sequencing data is also available from the European Nucleotide Archive. Note that this file will contain abundant errors in comparison to a consensus base call from high coverage data.E-VD-1838_rep1_all.fa.gzGe_arctos_Georgia_haploidised_fastaHaploidised fasta sequence generated by mapping Illumina short reads to the reference genome assembly of the giant panda, then randomly selecting a single high quality nucleotide from the read stack for each position of the reference genome. See the original publication for full details. The raw sequencing data is also available from the European Nucleotide Archive. Note that this file will contain abundant errors in comparison to a consensus base call from high coverage data.Ge_rep1_all.fa.gzGS136_ingressus_Austria_haploidised_fastaHaploidised fasta sequence generated by mapping Illumina short reads to the reference genome assembly of the giant panda, then randomly selecting a single high quality nucleotide from the read stack for each position of the reference genome. See the original publication for full details. The raw sequencing data is also available from the European Nucleotide Archive. Note that this file will contain abundant errors in comparison to a consensus base call from high coverage data.GS136_rep1_all.fa.gzHV74_kudarensis_Armenia_haploidised_fastaHaploidised fasta sequence generated by mapping Illumina short reads to the reference genome assembly of the giant panda, then randomly selecting a single high quality nucleotide from the read stack for each position of the reference genome. See the original publication for full details. The raw sequencing data is also available from the European Nucleotide Archive. Note that this file will contain abundant errors in comparison to a consensus base call from high coverage data.HV74_rep1_all.fa.gzLS039_arctos_Spain_haploidised_fastaHaploidised fasta sequence generated by mapping Illumina short reads to the reference genome assembly of the giant panda, then randomly selecting a single high quality nucleotide from the read stack for each position of the reference genome. See the original publication for full details. The raw sequencing data is also available from the European Nucleotide Archive. Note that this file will contain abundant errors in comparison to a consensus base call from high coverage data.LS039_rep1_all.fa.gzNB_maritimus_haploidised_fastaHaploidised fasta sequence generated by mapping Illumina short reads to the reference genome assembly of the giant panda, then randomly selecting a single high quality nucleotide from the read stack for each position of the reference genome. See the original publication for full details. The raw sequencing data is also available from the European Nucleotide Archive. Note that this file will contain abundant errors in comparison to a consensus base call from high coverage data.NB_rep1_all.fa.gzSB_maritimus_haploidised_fastaHaploidised fasta sequence generated by mapping Illumina short reads to the reference genome assembly of the giant panda, then randomly selecting a single high quality nucleotide from the read stack for each position of the reference genome. See the original publication for full details. The raw sequencing data is also available from the European Nucleotide Archive. Note that this file will contain abundant errors in comparison to a consensus base call from high coverage data.SB_rep1_all.fa.gzSwe_arctos_Sweden_haploidised_fastaHaploidised fasta sequence generated by mapping Illumina short reads to the reference genome assembly of the giant panda, then randomly selecting a single high quality nucleotide from the read stack for each position of the reference genome. See the original publication for full details. The raw sequencing data is also available from the European Nucleotide Archive. Note that this file will contain abundant errors in comparison to a consensus base call from high coverage data.Swe_rep1_all.fa.gzTornatus_haploidised_fastaHaploidised fasta sequence generated by mapping Illumina short reads to the reference genome assembly of the giant panda, then randomly selecting a single high quality nucleotide from the read stack for each position of the reference genome. See the original publication for full details. The raw sequencing data is also available from the European Nucleotide Archive. Note that this file will contain abundant errors in comparison to a consensus base call from high coverage data.Tornatus_all.fa.gzUamericanus_haploidised_fastaHaploidised fasta sequence generated by mapping Illumina short reads to the reference genome assembly of the giant panda, then randomly selecting a single high quality nucleotide from the read stack for each position of the reference genome. See the original publication for full details. The raw sequencing data is also available from the European Nucleotide Archive. Note that this file will contain abundant errors in comparison to a consensus base call from high coverage data.Uamericanus_all.fa.gzUap_arctos_Pleistocene_Austria_haploidised_fastaHaploidised fasta sequence generated by mapping Illumina short reads to the reference genome assembly of the giant panda, then randomly selecting a single high quality nucleotide from the read stack for each position of the reference genome. See the original publication for full details. The raw sequencing data is also available from the European Nucleotide Archive. Note that this file will contain abundant errors in comparison to a consensus base call from high coverage data.Uap_rep1_all.fa.gzUthibetanus_haploidised_fastaHaploidised fasta sequence generated by mapping Illumina short reads to the reference genome assembly of the giant panda, then randomly selecting a single high quality nucleotide from the read stack for each position of the reference genome. See the original publication for full details. The raw sequencing data is also available from the European Nucleotide Archive. Note that this file will contain abundant errors in comparison to a consensus base call from high coverage data.Uthibetanus_all.fa.gzWH2_maritimus_haploidised_fastaHaploidised fasta sequence generated by mapping Illumina short reads to the reference genome assembly of the giant panda, then randomly selecting a single high quality nucleotide from the read stack for each position of the reference genome. See the original publication for full details. The raw sequencing data is also available from the European Nucleotide Archive. Note that this file will contain abundant errors in comparison to a consensus base call from high coverage data.WH2_rep1_all.fa.gzWK01_eremus_Austria_haploidised_fastaHaploidised fasta sequence generated by mapping Illumina short reads to the reference genome assembly of the giant panda, then randomly selecting a single high quality nucleotide from the read stack for each position of the reference genome. See the original publication for full details. The raw sequencing data is also available from the European Nucleotide Archive. Note that this file will contain abundant errors in comparison to a consensus base call from high coverage data.WK01_rep1_all.fa.gz', 'descriptionType': 'Other'}], 'geoLocations': [], 'fundingReferences': [], 'url': 'http://datadryad.org/stash/dataset/doi:10.5061/dryad.cr1496b', 'contentUrl': None, 'metadataVersion': 26, 'schemaVersion': 'http://datacite.org/schema/kernel-4', 'source': 'mds', 'isActive': True, 'state': 'findable', 'reason': None, 'viewCount': 136, 'downloadCount': 25, 'referenceCount': 1, 'citationCount': 1, 'partCount': 18, 'partOfCount': 0, 'versionCount': 0, 'versionOfCount': 0, 'created': '2018-07-12T08:49:16Z', 'registered': '2018-07-12T08:49:18Z', 'published': None, 'updated': '2022-05-31T20:46:01Z'}, 'relationships': {'client': {'data': {'id': 'dryad.dryad', 'type': 'clients'}}}}], 'meta': {'total': 6, 'totalPages': 1, 'page': 1, 'states': [{'id': 'findable', 'title': 'Findable', 'count': 6}], 'resourceTypes': [{'id': 'dataset', 'title': 'Dataset', 'count': 6}], 'created': [{'id': '2022', 'title': '2022', 'count': 1}, {'id': '2020', 'title': '2020', 'count': 1}, {'id': '2018', 'title': '2018', 'count': 2}, {'id': '2017', 'title': '2017', 'count': 1}, {'id': '2016', 'title': '2016', 'count': 1}], 'published': [{'id': '2022', 'title': '2022', 'count': 1}, {'id': '2020', 'title': '2020', 'count': 1}, {'id': '2019', 'title': '2019', 'count': 1}, {'id': '2018', 'title': '2018', 'count': 2}, {'id': '2016', 'title': '2016', 'count': 1}], 'registered': [{'id': '2022', 'title': '2022', 'count': 1}, {'id': '2020', 'title': '2020', 'count': 1}, {'id': '2018', 'title': '2018', 'count': 2}, {'id': '2017', 'title': '2017', 'count': 1}, {'id': '2016', 'title': '2016', 'count': 1}], 'providers': [{'id': 'dryad', 'title': 'Dryad', 'count': 5}, {'id': 'bgaq', 'title': 'UC Davis', 'count': 1}], 'clients': [{'id': 'dryad.dryad', 'title': 'DRYAD', 'count': 5}, {'id': 'cdl.ucd', 'title': 'UC Davis', 'count': 1}], 'affiliations': [{'id': 'ror.org/051qn8h41', 'title': 'Ilia State University', 'count': 6}, {'id': 'ror.org/02kkvpp62', 'title': 'Technical University Munich', 'count': 2}, {'id': 'ror.org/057qpr032', 'title': 'University of Bordeaux', 'count': 2}, {'id': 'ror.org/0078xmk34', 'title': 'University of Montana', 'count': 1}, {'id': 'ror.org/00cv9y106', 'title': 'Ghent University', 'count': 1}, {'id': 'ror.org/00j75pt62', 'title': 'Technical University of Zvolen', 'count': 1}, {'id': 'ror.org/00rwcdf75', 'title': 'Ukrainian National Forestry University', 'count': 1}, {'id': 'ror.org/00s67c790', 'title': 'Estonian University of Life Sciences', 'count': 1}, {'id': 'ror.org/011q66e29', 'title': 'Instituto Nacional de Investigación y Tecnología Agraria y Alimentaria', 'count': 1}, {'id': 'ror.org/012dxyr07', 'title': 'University of Agriculture in Krakow', 'count': 1}], 'prefixes': [{'id': '10.5061', 'title': '10.5061', 'count': 5}, {'id': '10.25338', 'title': '10.25338', 'count': 1}], 'certificates': [], 'licenses': [{'id': 'cc0-1.0', 'title': 'CC0-1.0', 'count': 6}], 'schemaVersions': [{'id': '4', 'title': 'Schema 4', 'count': 6}], 'linkChecksStatus': [], 'subjects': [{'id': 'overyielding', 'title': 'Overyielding', 'count': 2}, {'id': 'Bartonella', 'title': 'Bartonella', 'count': 1}, {'id': 'Basal area growth', 'title': 'Basal Area Growth', 'count': 1}, {'id': 'Biogeographical field experiments', 'title': 'Biogeographical Field Experiments', 'count': 1}, {'id': 'Centaurea solstitialis', 'title': 'Centaurea Solstitialis', 'count': 1}, {'id': 'Dogs', 'title': 'Dogs', 'count': 1}, {'id': 'FOS: Agriculture, forestry, and fisheries', 'title': 'Fos: Agriculture, Forestry, And Fisheries', 'count': 1}, {'id': 'Fagus sylvatica', 'title': 'Fagus Sylvatica', 'count': 1}, {'id': 'Genetic loci', 'title': 'Genetic Loci', 'count': 1}, {'id': 'Holocene', 'title': 'Holocene', 'count': 1}], 'fieldsOfScience': [], 'citations': [{'id': '2022', 'title': '2022', 'count': 1}, {'id': '2019', 'title': '2019', 'count': 1}, {'id': '2018', 'title': '2018', 'count': 3}, {'id': '2016', 'title': '2016', 'count': 1}], 'views': [{'id': '2020', 'title': '2020', 'count': 35}, {'id': '2019', 'title': '2019', 'count': 136}, {'id': '2018', 'title': '2018', 'count': 309}, {'id': '2016', 'title': '2016', 'count': 71}], 'downloads': [{'id': '2020', 'title': '2020', 'count': 7}, {'id': '2019', 'title': '2019', 'count': 25}, {'id': '2018', 'title': '2018', 'count': 103}, {'id': '2016', 'title': '2016', 'count': 14}]}, 'links': {'self': 'https://api.datacite.org/dois?query=creators.affiliation.affiliationIdentifier:%22https://ror.org/051qn8h41%22'}}\n"
     ]
    }
   ],
   "source": [
    "# import libraries\n",
    "import json\n",
    "import urllib.request, json \n",
    "\n",
    "# connect to DataCite REST API and get items with special ROR ID and type dataset\n",
    "with urllib.request.urlopen('https://api.datacite.org/dois?query=creators.affiliation.affiliationIdentifier:\"https://ror.org/051qn8h41\"') as url:\n",
    "    \n",
    "    # create a json file out of API call\n",
    "    DataCite = json.load(url)\n",
    "    \n",
    "    # print json in Jupyter Notebook\n",
    "    print(DataCite)\n",
    "\n",
    "# create and open a new json file    \n",
    "with open('DataCite_Metadata_ROR051qn8h41.json', 'w') as f:\n",
    "    # write results in json file\n",
    "    json.dump(DataCite, f)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "51f670ed-7676-464c-93ba-dd531435970b",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.5"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
